source/configuration/modules/imfile.rst

```diff
@@ -1,570 +1,408 @@
-   single: imfile; readMode
-.. function:: readMode [mode]
-
-   This provides support for processing some standard types of multiline
-   messages. It is less flexible than ``startmsg.regex`` but offers higher
-   performance than regex processing. Note that ``readMode`` and
-   ``startmsg.regex`` cannot both be defined for the same input.
-
-   The value can range from 0-2 and determines the multiline
-   detection method.
-
-   0 - (**default**) line based (each line is a new message)
-
-   1 - paragraph (There is a blank line between log messages)
-
-   2 - indented (new log messages start at the beginning of a line. If a
-   line starts with a space or tab "\t" it is part of the log message before it)
-
-.. index:: 
-   single: imfile; escapeLF
-.. function:: escapeLF [on/off] (requires v7.5.3+)
-
-   This is only meaningful if multi-line messages are to be processed.
-   LF characters embedded into syslog messages cause a lot of trouble,
-   as most tools and even the legacy syslog TCP protocol do not expect
-   these. If set to "on", this option avoid this trouble by properly
-   escaping LF characters to the 4-byte sequence "#012". This is
-   consistent with other rsyslog control character escaping. By default,
-   escaping is turned on. If you turn it off, make sure you test very
-   carefully with all associated tools. Please note that if you intend
-   to use plain TCP syslog with embedded LF characters, you need to
-   enable octet-counted framing.
-   For more details, see Rainer's blog posting on imfile LF escaping. 
-
-.. index:: 
-   single: imfile; MaxLinesAtOnce
-.. function:: MaxLinesAtOnce [number]
-
-   This is a legacy setting that only is supported in *polling* mode.
-   In *inotify* mode, it is fixed at 0 and all attempts to configure
-   a different value will be ignored, but will generate an error
-   message.
-
-   Please note that future versions of imfile may not support this
-   parameter at all. So it is suggested to not use it.
-
-   In *polling* mode, if set to 0, each file will be fully processed and
-   then processing switches to the next file. If it is set to any other
-   value, a maximum of [number] lines is processed in sequence for each file,
-   and then the file is switched. This provides a kind of mutiplexing
-   the load of multiple files and probably leads to a more natural
-   distribution of events when multiple busy files are monitored. For
-   *polling* mode, the **default** is 10240.
-
-.. index:: 
-   single: imfile; MaxSubmitAtOnce
-.. function:: MaxSubmitAtOnce [number]
-
-   This is an expert option. It can be used to set the maximum input
-   batch size that imfile can generate. The **default** is 1024, which
-   is suitable for a wide range of applications. Be sure to understand
-   rsyslog message batch processing before you modify this option. If
-   you do not know what this doc here talks about, this is a good
-   indication that you should NOT modify the default.
+*****************
```

Comment:
I don't mind, just document this somewhere so that others will do the same (I generally wouldn't bother with different types of characters, but that's part of the reason I'm not the best person to write docs :-)